Model/Consumer.php

```diff
@@ -42,56 +54,61 @@ public function __construct(
      */
     public function process($request)
     {
-        try {
-            $data = json_decode($request, true);
-            $storeId = $data['storeId'] ?? null;
-            $module = $data['module'];
-            $entity = $data['entity'];
-            $key = $data['key'];
-            $value = $data['value'];
-            $type = $data['type'];
-            $refund = $data['refund'] ?? false;
+        $data = json_decode($request, true);
+        if ($data['type'] == 'disconnect') {
+            $this->productsHelper->cleanProductStorekeeperId($data['storeId']);
+            $this->authHelper->disconnectStore($data['storeId']);
+        } else {
+            try {
```

Comment:
```suggestion
            $this->processEvent()
```
Maybe good to extract it below, than also my PR will be cleaner :P